mava/systems/ippo/networks.py

```diff
@@ -174,10 +176,19 @@ class ValueHead(hk.Module):
     def __init__(
         self,
         name: Optional[str] = None,
+        w_init: Optional[hk.initializers.Initializer] = None,
     ):
-        """Initialize the class"""
+        """Initialize the value head.
+
+        Args:
+            name: An optional string name for the class. Must be a valid
+                Python identifier. If ``name`` is not provided then the class
+                name for the current instance is converted to
+                ``lower_snake_case`` and used instead.
```

Comment:
Lower_snake_case? 🤔

Reply:
I got this from Haiku's docstrings :sweat_smile: 
I think it refers to converting `SomeClassName` to `some_class_name` when no name parameter is given when the class is initialised. Happy to remove this or change it. :)

Reply:
Totally agree with this. I've updated the docstring accordingly :)